### PR TITLE
[4.0] Update RandomImageHelper.php

### PR DESCRIPTION
**Pull request by HRIT-Florian**
Fix for #22065 and improved code to scaled to the specified final height or width (so that the final height or width is not exceeded and there is no loss in dimension) why wrong complex, If it's that simple ;)

Pull Request for Issue #22065 .

### Testing Instructions
See steps: #22065
Try once with custom height, once with custom width.
Once with both and once with empty height and width.

### Expected result
Final height or width should be calculated in dimension even if no width or hight is set
Check px specification in width or heigt attibute (Not the given look, if huge px, because of box size limits possible ;)

### Actual result
Works a bit complex  without dimenson loss but it works only if width or hight is set!

### Documentation Changes Required
No

### CODE REVIEW

File: modules/mod_random_image/Helper/RandomImageHelper.php [hunk 1]
```diff
@@ -45,38 +45,18 @@ public static function getRandomImage(&$params, $images)
...
-				$width = $height * $coeff;
-			}
+			$width = $size[0] / $size[1] * $height;
 		}
 
-		$image->width  = $width;
-		$image->height = $height;
+		$image->width  = (int) $width;
+		$image->height = (int) $height;
+
```
Comment (Quy) [inline modules/mod_random_image/Helper/RandomImageHelper.php]: Remove tabs on line 59.

Comment (HRIT-Florian) [reply modules/mod_random_image/Helper/RandomImageHelper.php]: done

**Comment by roland-d** [comment]
@Didldu-Florian Could you please resolve the merge conflict so I can get this tested?

**Comment by Quy** [comment]
Please fix conflicting file.

**Comment by Quy** [comment]
Closing due to no response.

**Pull request closed without merging**